src/App.js

```diff
@@ -4,94 +4,131 @@
  *              Login Gate will be displayed upon Auth.
  */
 
-import React, { useState, useEffect } from 'react';
+import React, { useState, useEffect, useContext } from 'react';
 import './App.css';
-import DefaultHeader from "./containers/DefaultLayout/DefaultHeader";
-import DefaultLayout from "./containers/DefaultLayout/DefaultLayout";
-import LoginGate from "./views/Login/LoginGate";
 import {LoggingContext} from "./context/LoggingContext";
 import {useLogger} from "./hooks/useLogger";
-import {Col} from "reactstrap";
+import {Col, Row} from "reactstrap";
 import {VoiceCommandsContext} from "./context/VoiceCommandsContext";
 import useSpeechRecognition from "./hooks/useSpeechRecognition";
-import TestPage from "./views/TestPage/TestPage";
-import {BrowserRouter as Router, Link, Redirect, Route, Switch} from "react-router-dom";
 import {Stitch} from "mongodb-stitch-browser-sdk";
 import {useMongo} from "./hooks/useMongo";
 import {AppContext} from "./context/AppContext";
-import Login from "./views/Login/Login";
+import Routing from "./containers/Routing/Routing";
 
-/**
- *
- * @returns {*}
- * @constructor
- */
 const App = () => {
 
     const [client, setClient] = useState(null);
+    const [showLogger, setShowLogger] = useState(false);
+    const [showTranscript, setShowTranscript] = useState(false);
+    const [showIntendArray, setShowIntendArray] = useState(false);
 
-    const logger = useLogger(["App Initialized"]);
+    const logger = useLogger(["App Initialized"], setShowLogger);
```

Comment:
Need to remove this `setShowLogger` param, was something I was experimenting with